test/blackbox-tests/test-cases/pkg/opam-var/opam-var-os.t

```diff
@@ -0,0 +1,19 @@
+  $ . ./opam-var-helpers.sh
+
+Here we test global opam variables that are system specific. Since these values change
+between systems, we can't hardcode them in the test. Instead, we use the opam var command
+to compare their values.
+
+# arch os os-distribution os-family os-version user group
+
+We write all the dune values to a file and then diff them with the output of opam var.
+
+  $ test arch os os-distribution os-family os-version >/dev/null 2> dune-vars 
+  [1]
```

Comment:
This fails for the moment as the os variables don't all work during the translation step. This will be fixed by #8738.